src/subscription_manager/gui/firstboot/rhsm_login.py

```diff
@@ -216,6 +217,9 @@ def shouldAppear(self):
         identity = inj.require(inj.IDENTITY)
         return not identity.is_valid()
 
+    def on_register_message(self, obj, msg, message_type=ga_Gtk.MessageType.ERROR):
```

Comment:
I'd avoid setting a arg to a non-None default. Those get evaluated at module import time and can
cause a variety of assorted unexpected behavior. In this particular case, the value is a reference to
a constant so shouldn't cause too much chaos, but I'd rather avoid it in general.

``` python
def on_register_message(self, obj, msg, msg_type=None):
    msg_type = msg_type or ga_Gtk.MessageType.ERROR
    self.error_dialog(msg, msg_type=msg_type)
```

(Not sure what the default msg_type should be, I lean towards INFO, but since this is specific to our firstboot
module, ERROR may make more sense as default)